test/python/utils.py

```diff
@@ -20,11 +28,271 @@ def tarantool_version():
+            continue
+
+        filemode = rpm.headers['filemodes'][i]
+        assert_filemode(filepath, filemode)
+
+
+def assert_tarantool_dependency_deb(filename):
+    with open(filename) as control:
+        control_info = control.read()
+
+        depends_str = re.search('Depends: (.*)', control_info)
+        assert depends_str is not None
+
+        min_version = re.findall(r'\d+\.\d+\.\d+', tarantool_version())[0]
+        max_version = str(int(re.findall(r'\d+', tarantool_version())[0]) + 1)
+
+        deps = depends_str.group(1)
+        assert 'tarantool (>= {})'.format(min_version) in deps
+        assert 'tarantool (<< {})'.format(max_version) in deps
+
+
+def assert_tarantool_dependency_rpm(filename):
+    rpm = rpmfile.open(filename)
+    dependency_keys = ['requirename', 'requireversion', 'requireflags']
+    for key in dependency_keys:
+        assert key in rpm.headers
+
+    assert len(rpm.headers['requirename']) == 2
+    assert len(rpm.headers['requireversion']) == 2
+    assert len(rpm.headers['requireversion']) == 2
+
+    min_version = re.findall(r'\d+\.\d+\.\d+', tarantool_version())[0]
+    max_version = str(int(re.findall(r'\d+', tarantool_version())[0]) + 1)
+
+    assert rpm.headers['requirename'][0].decode('ascii') == 'tarantool'
+    assert rpm.headers['requireversion'][0].decode('ascii') == min_version
+    assert rpm.headers['requireflags'][0] == 0x08 | 0x04  # >=
+
+    assert rpm.headers['requirename'][1].decode('ascii') == 'tarantool'
+    assert rpm.headers['requireversion'][1].decode('ascii') == max_version
+    assert rpm.headers['requireflags'][1] == 0x02  # <
+
+
+def check_systemd_dir(basedir):
+    systemd_dir = (os.path.join(basedir, 'etc/systemd/system'))
+    assert os.path.exists(systemd_dir)
+
+    systemd_files = recursive_listdir(systemd_dir)
+
+    assert len(systemd_files) == 2
+    assert '{}.service'.format(project_name) in systemd_files
+    assert '{}@.service'.format(project_name) in systemd_files
+
+
+def check_package_files(basedir, project_path):
+    # check if only theese files are delivered
+    for filename in recursive_listdir(basedir):
+        assert any([
+            filename.startswith(prefix) or prefix.startswith(filename)
+            for prefix in [
+                os.path.join('usr/share/tarantool', project_name),
+                'etc/systemd/system',
+                'usr/lib/tmpfiles.d'
+            ]
+        ])
+
+    # check distribution dir content
+    distribution_dir = os.path.join(basedir, 'usr/share/tarantool', project_name)
+    assert os.path.exists(distribution_dir)
+    assert_dir_contents(recursive_listdir(distribution_dir))
+
+    # check systemd dir content
+    check_systemd_dir(basedir)
+
+    # check tmpfiles conf
+    project_tmpfiles_conf_file = os.path.join(basedir, 'usr/lib/tmpfiles.d', '%s.conf' % project_name)
+    assert open(project_tmpfiles_conf_file).read().find('d /var/run/tarantool') != -1
```

Comment:
You don't close a file here

Reply:
O, thanks! This fix was moved to the #91 PR, I will move it here.